net/sync/model/src/main/java/de/danoeh/antennapod/net/sync/model/EpisodeAction.java

```diff
@@ -72,6 +74,10 @@ public static EpisodeAction readFromJsonObject(JSONObject object) {
                 e.printStackTrace();
             }
         }
+        String guid = object.optString("guid", null);
```

Comment:
I think it would be better to use `episode_guid`. The reason is that there is now the `podcast:guid` tag and explicitly writing "episode" makes it more specific which guid is used.

Reply:
this is saved within the EpisodeAction entity thus belongs to Episode(Action). If i'd save a podcast guid in that table i'd prefix it as such. 
With the same argument i'd need to prefix the episode link/url with _episode_  since the podcast also has a link.

Reply:
I updated the PR to use `guid`. Let's see if they merge it. Gpodder.net seems to be pretty much dead to me.